osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/configuration/service/ProtocolServiceLookup.java

```diff
@@ -0,0 +1,31 @@
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.configuration.service;
+
+import java.util.List;
+
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.entities.Protocol;
+import org.opensmartgridplatform.adapter.protocol.dlms.exceptions.ProtocolAdapterException;
+
+public class ProtocolServiceLookup {
+
+    private final List<ProtocolService> protocolServices;
+
+    public ProtocolServiceLookup(final List<ProtocolService> protocolServices) {
```

Comment:
I looked at the code, but this class is never instantiated, only in tests

Reply:
It is injected by Spring.

Reply:
but where? ProtocolServiceLookup is not a Spring bean, and the user GetConfigurationObjectCommandExecutor has no autowired constructor nor does it autowire this lookup class.

Reply:
You are right, I forgot the @Component annotation on the ProtocolServiceLookup. Has been discovered and fixed by me in the meantime.